src/test/resources/data/InstructorStudentListPageUiTest.json

```diff
@@ -67,6 +63,7 @@
       "name": "Instructor1 Course1",
       "email": "instr@course1.tmt",
       "role":"Co-owner",
+      "isArchived":false,
       "isDisplayedToStudents":true,
       "displayedName":Instructor",
```

Comment:
Is there a missing double quote before `Instructor` here? Might be what's causing GitHub to display everything else in red.